### ISSUE TITLE
Previews take long time

### ISSUE THREAD

**Issue by dawid3294443** [TheLastBen/fast-stable-diffusion#1808]
After generating image(s), the preview takes a really long time to load (about 30 seconds/4 images). Is it normal? Can something be done to make this faster?

**Comment by TheLastBen** [comment]
what resolution ?

**Comment by dawid3294443** [comment]
512x768 4 images, so 1024x1572

**Comment by dawid3294443** [comment]
No, generation time is OK. But after the images are generated, they are shown on the right sight of webui. And this is where I have a problem, it takes a lot of time for them to fully "load".

**Comment by TheLastBen** [comment]
I think you should check your connection latency

**Comment by dawid3294443** [comment]
I've got 300mb/s internet, no problem with anything else
